### PR TITLE
Updated `fury versions` to include package size and date updated.

### PR DESCRIPTION
**Pull request by markjeee**
Output looks like this:

```
*** Cableguy Versions ***

0.5.2.alpha.1.g83c3230 (24576 bytes, 2018-10-08 10:00 +0000)
0.5.1.pre14 (24576 bytes, 2018-04-17 12:51 +0000)
```

### CODE REVIEW

File: lib/gemfury/client.rb [hunk 1]
```diff
@@ -41,6 +41,13 @@ def versions(name, options = {})
       checked_response_body(response)
     end
 
+    def package(name, options = {})
+      ensure_ready!(:authorization)
+      url = "packages/#{escape(name)}"
+      response = connection(:api_version => 2).get(url, options)
```
Comment (rykov) [inline lib/gemfury/client.rb]: API v2 is not stable and shouldn't be used in the CLI. This particular endpoint, for example, is deprecated and is no longer used by the Dashboard. Better to add the needed details to API v1.

File: lib/gemfury/command/app.rb [hunk 2]
```diff
@@ -37,10 +39,13 @@ def list
...
     with_checks_and_rescues do
-      versions = client.versions(gem_name)
+      package = client.package(gem_name)
+      versions = package['versions']
       shell.say "\n*** #{gem_name.capitalize} Versions ***\n\n"
       versions.each do |v|
-        shell.say v['version']
+        size = '%s bytes' % v['size']
+        uploaded = Time.parse(v['updated_at']).strftime('%F %R %z')
+        shell.say "#{v['version']} (#{size}, #{uploaded})"
```
Comment (rykov) [inline lib/gemfury/command/app.rb]: Is there a Thor or Highline feature to print things aligned in table-like form, rather than doing parentheses?

**Comment by markjeee** [comment]
[`fury versions` show size of package and date uploaded.](https://trello.com/c/P3XuqHBE/4-fury-versions-show-size-of-package-and-date-uploaded)

File: lib/gemfury/command/app.rb [hunk 3]
```diff
@@ -1,3 +1,5 @@
+require 'time'
```
Comment (rykov) [inline lib/gemfury/command/app.rb]: This should go into `lib/gemfury.rb`

File: lib/gemfury/command/app.rb [hunk 4]
```diff
@@ -40,9 +42,16 @@ def versions(gem_name)
     with_checks_and_rescues do
       versions = client.versions(gem_name)
       shell.say "\n*** #{gem_name.capitalize} Versions ***\n\n"
+
+      va = []
+      va << [ 'version', 'authors', 'uploaded' ]
```
Comment (rykov) [inline lib/gemfury/command/app.rb]: Why not collapse these two lines into:

    va = %w(version authors uploaded)

File: lib/gemfury/command/app.rb [hunk 5]
```diff
@@ -40,9 +42,16 @@ def versions(gem_name)
...
       versions.each do |v|
-        shell.say v['version']
+        authors = v['authors'].join(', ')
+        uploaded = Time.parse(v['updated_at']).strftime('%F %R %z')
```
Comment (rykov) [inline lib/gemfury/command/app.rb]: `'authors'` is just package metadata. To match the dashboard, you'll need to switch that to 'created_by' (need to expose this via API first). Also, we use `created_at  for "Uploaded at" on the dashboard, not `updated_at`.

**Comment by rykov** [comment]
@markjeee I've added `created_by` to the API. Please note that the field can be `null` because some older versions were created before we collected this information.

**Pull request merged**